### DISCUSSION TITLE
Paging to get records, how to get the total number of records with /api/2.0

### DISCUSSION THREAD

**Discussion by fzwu** [goharbor/harbor#17812] [Q&A]
I need to obtain records through API. After obtaining records according to the paging interface provided by API, there is no total number of records returned, so I cannot calculate whether there is a next page and the total number of records.
eg:
api/v2.0/labels?sort=-creation_time&scope=g&project_id=0&page_size=15&page=1
This API returns a list of records and cannot get data related to paging.
How can I get paging data?

**Comment by AllForNothing** [comment]
You get the total count from the response header ` x-total-count`.

We recommend using Harbor API center when your want to learn or try Harbor APIs:

1. Open  `${your_harbor_domain}/devcenter-api-2.0` to visit Harbor API center.
![image](https://user-images.githubusercontent.com/30999793/203013033-e202f6a1-fd45-42da-8ced-339d53d78cc0.png)

2. Set account and password(skip it If you have logged in already)
![image](https://user-images.githubusercontent.com/30999793/203013506-5ff8dd4e-ec9c-4a14-8358-0b3a446455a9.png)

3. Expand  a certain API,  and click `Try it out`
![image](https://user-images.githubusercontent.com/30999793/203014141-2196bc1b-8ac0-41ce-848e-71b05192dcce.png)

4. For `post/patch/update` requests, execute it again if you get a CSRF token invalid error
5.  View the response.
![image](https://user-images.githubusercontent.com/30999793/203014737-e64ef4ea-180f-48a7-8367-7866a4de0501.png)
